content/en/apps/guides/offline/_index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Offline CHT Server"
+linkTitle: "Offline CHT"
+weight: 100
+description: >
+ Running an instance of CHT Core server with no internet connection
+---
+
+## Introduction
+
+The CHT is built as an offline first application. This applies to clients, either  browsers or Android applications, connecting to the CHT server.  The server itself assumes it has Internet connectivity to provide services such as DNS, software updates and general use connectivity.  This document explores what it looks like when the CHT server is offline without these services available.
+
+Running a CHT server offline requires no modifications to the CHT itself.  Instead, supporting services normally found online are replicated locally. 
+
+## Considerations
+
+An offline CHT server is most appropriate for a development environment.  There are serious implications to consider before deploying an offline server in a production environment which are most often ignored for development.  Offline solution should have a plan in place for:
+
+* Alerting - How will alerts be sent in the case of downtime or degraded service? 
+* Power failures and unplanned restarts - Will the server cleanly restart such that the CHT resumes service correctly?
+* Backups - What happens to the CHT data if there's a hard drive failure?  
```

Comment:
I think we can revamp our cht-core deploy documentation and link to that for how to handle these considerations on the main app. But this document should have advice on how to handle backup of configs, failovers, reinstalls and migrating the dns to another setup.

Reply:
Yes, I'd love to link to more documentation for any of these bullet list items here!  Please let me know when you have destinations to link to.

When you say:

> But this document should have advice on how to handle backup of configs, failovers, reinstalls and migrating the dns to another setup.

Do you mean "this" as in the offline CHT server doc or do you mean "this" as in revamped cht-core deploy documentation? If it's the latter I agree, if it's the former, I disagree.

Reply:
We agree!